src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/translation/KuntaApiPtvTranslator.java

```diff
@@ -314,6 +333,30 @@
       .collect(Collectors.toList());
   }
 
+  /**
+   * Translates Kunta API WebPages into PTV language items
+   * 
+   * @param webPages Kunta API WebPages
+   * @return PTV language items
+   */
+  public List<VmOpenApiLanguageItem> translateWebPagesIntoLanguageItems(List<WebPage> webPages) {
+    if (webPages == null) {
+      return Collections.emptyList();
+    }
+    
+    return webPages.stream()
+      .filter((webPage) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "webPage" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)